java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerIT.java

```diff
@@ -65,13 +70,18 @@ public void beforeClass(@org.testng.annotations.Optional Boolean usingEncryption
         }
         mantaClient = new MantaClient(config);
 
+        // sanity check
```

Comment:
Verify that user credentials are valid by checking for existence of home folder. This will prevent us from skipping the test suite because we assume MPU is not available on the Manta instance used for running the tests.